docs-source/content/security/secrets.md

```diff
@@ -111,43 +98,40 @@ $ helm install my-weblogic-operator kubernetes/charts/weblogic-operator \
 
 {{% notice info %}}
 For more information, see
-[Install the operator Helm chart]({{<relref "/userguide/managing-operators/installation/_index.md#install-the-operator-helm-chart">}})
-under **User Guide**.
+[Install the operator Helm chart]({{<relref "/userguide/managing-operators/installation/_index.md#install-the-operator-helm-chart">}}).
 {{% /notice %}}
 
-#### WebLogic Kubernetes Operator configuration override secrets
+#### Operator configuration override secrets
```

Comment:
section title --> Operator configuration secrets

Below make it clear that override usage for configuration.secrets is specific to DiI and DiPV.  

Plus add mention MiI use of configuration.secrets via WDT SECRET macros as documented in model-in-image/model-files.md and runtime-updates.md)